examples/js-demo/index.html

```diff
@@ -16,6 +16,7 @@ <h1>Algolia Recommendations</h1>
 
       <div id="autocomplete"></div>
 
+      <div id="slider"></div>
```

Comment:
We don't need this container in the demo.

Reply:
Oops I forgot this one: fixed in https://github.com/algolia/ui-components/pull/28/commits/3e49a8bf3105aba0cca8baa4d96b5741d49df393